com/uber/kafkaSpraynozzle/stats/StatsDReporter.java

```diff
@@ -0,0 +1,61 @@
+package com.uber.kafkaSpraynozzle.stats;
+
+import com.timgroup.statsd.NonBlockingStatsDClient;
+import com.timgroup.statsd.StatsDClient;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * Statsd StatsReporter implementation
+ *
+ * To initialize this class you need to provide it the path to a .properties file containing:
```

Comment:
Yeah, i went back and forth on this... initially i was passing a StatsD specific json blob in via the comand line, but that got ugly!  I felt like this was the cleanest approach to take, but please let me know if you can think of something else.